docs/modules/ROOT/pages/references/cp_node_scaling.adoc

```diff
@@ -0,0 +1,51 @@
+= Control Plane Node Scaling
+
+The control plane node resource requirements depent on a number of different metrics.
+CPU requirements generally depend on the ammount on API write requests,
+while memory requirements generally depend on the amount of resource objects (most importantly Pod count) in the cluster.
+
+To avoid cascading failures during a scheduled node reboot or an unexpected node failure,
+it's recommended to keep the overall CPU and memory usage not significantly higher than 60%.
+
+== Guidelines
+
+The following guiedlines are an estimate, based on metrics collected from our managed clusters.
+Individual clusters may vary depending on worker node size and individual workload running on the cluster.
```

Comment:
I'd add a note that depending on cloud provider the suggested sizing may not be a supported VM size.

```suggestion
Individual clusters may vary depending on worker node size and individual workload running on the cluster.

[NOTE]
====
Many cloud providers don't support arbitrary VM sizes, so we'll usually recommend the closest available VM size that's at least as large as the recommended size below.
====
```